src/filesystem/nodes/node_data/mod.rs

```diff
@@ -240,6 +241,18 @@ impl NodeData {
         })
     }
 
+    pub(crate) fn empty_file() -> Self {
+        Self::File {
+            permissions: FilePermissions::default(),
+            associated_data: HashMap::new(),
+            content: FileContent::Encrypted {
+                locked_access_key: SymLockedAccessKey::default(),
+                cid: Cid::default(),
```

Comment:
The default all zeros isn't quite right here, the hash over an empty struct does has a specific value and I'd bet its not all zeros if we calculated it. We should probably calculate that and store it as another special Cid constant.